tests/install_upgrade_operators/console_cli_download/utils.py

```
@@ -82,3 +85,30 @@ def validate_custom_cli_downloads_urls_updated(
+        for sample in TimeoutSampler(
+            wait_timeout=TIMEOUT_2MIN,
+            sleep=TIMEOUT_10SEC,
+            func=download_and_extract_file_from_cluster,
+            exceptions_dict={requests.exceptions.HTTPError: []},
+            tmpdir=tmpdir,
+            url=url,
+        ):
+            if sample:
+                return sample
+    except TimeoutExpiredError:
+        LOGGER.error(f"Failed to download custom CLI archive from {url} after retrying on HTTP errors")
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)